Week3/README/READEME.md

```diff
@@ -0,0 +1,56 @@
+<h1>Run and Test the project:</h1>
+
+<h3> Clone repository:</h3>
+
+`https://github.com/ShyarK/databases.git`
+
+`npm i`
+
+`npm start`
+
+<h3> To get all users:</h3>
+
+`http://localhost:3000/users`
+
+<h3> To get all todolist:</h3>
+
+`http://localhost:3000/todolist`
+
+<h3> To get all todo items:</h3>
+
+`http://localhost:3000/todoitems`
+
+<h3> To get one user with the given id:</h3>
+
+`http://localhost:3000/user/7`
+
+<h3> To get one todoList with the given id:</h3>
+
+`http://localhost:3000/todolist/2`
```

Comment:
Usually you present the doc like:
http://localhost:3000/todolist/{id}, then you can add an example.

Reply:
I didn't know that we need to use ``{id}``, in the doc! As I know so far we use template literals or string literals to allow embedded expressions: 
For example: 
``const person = 'Mike';``
``const age = 28;``
``const output = myTag`That ${ person } is a ${ age }`;``

And also we use it ``react.js`` to embed expressions in JSX:
``const name = 'Josh Perez';``
``const element = <h1>Hello, {name}</h1>;``
You asked to use it in the doc file! Can you give an example so I can change it? Actually, this is my first time to write README.md file.

Reply:
You don't NEED to use but it's conventional. I'm not sure there is one standard but you can use parameters in {name} or :name .
Here is one of github api documentation:
https://developer.github.com/v3/orgs/members/

Usually documentation are generated from code by annotated the code itself.

The documentation won't affect your code, it's just a way to explain the user how to perform a call. Then you can provide example of call to help the developer understand.